module/User/src/User/Form/Register.php

```diff
@@ -69,6 +69,13 @@ public function setError($error)
                 ]
             ]);
             break;
+        case self::ERROR_ALREADY_REGISTERED:
```

Comment:
I know indentation is incorrect, but at least it's consistent with the other cases.